docs/related-tools/local-analysis/client-side-tools.md

```diff
@@ -16,6 +16,11 @@ Codacy supports client-side tools in two ways:
 
     The Codacy Analysis CLI automatically fetches the code pattern settings that you define on the Codacy UI and applies them when running these tools.
 
+## Running the client-side tools
+
+!!! tip
+    **If you're using GitHub** we recommend that you use the [Codacy GitHub Action](https://github.com/codacy/codacy-analysis-cli-action#integration-with-codacy-for-client-side-tools) to run the client-side tools and upload the results to Codacy.
```

Comment:
```suggestion
    **If you're using GitHub** we recommend that you use the [Codacy Analysis CLI GitHub Action](https://github.com/codacy/codacy-analysis-cli-action#integration-with-codacy-for-client-side-tools) to run the client-side tools and upload the results to Codacy.
```

this is its name in the Marketplace: https://github.com/marketplace/actions/codacy-analysis-cli
(not to be confused with the coverage one)

Reply:
Very good catch. :raised_hands: See https://github.com/codacy/codacy-analysis-cli-action/pull/49 for updating the name of the GitHub Action on its own README. :sweat_smile: